### DISCUSSION TITLE
Icons... bugged?

### DISCUSSION THREAD

**Discussion by 02xa** [Matt-FTW/dotfiles#21] [Q&A] [answered]
Hey, I'm using this dotfiles and for some reason this icons are in this way.
I want to know if is there a way to fix.
So, appreciate if someone can tell me!!

![image](https://github.com/user-attachments/assets/acf8ac7b-1327-416a-b924-a327bbe80d11)
![image](https://github.com/user-attachments/assets/32eea9ae-cdea-4b64-a80f-dd1f72f064ab)

**Comment by Matt-FTW** [comment, marked as answer]
Hey!

Go to the **fonts section** on the [installation](https://github.com/Matt-FTW/dotfiles?tab=readme-ov-file#package-dependencies-installation). Check you have every package installed and in their latest version.

**Reply by 02xa** [reply]
I tried this before and ain't worked.
For some reason the "ttf-ms-win11-auto" package gets an error in the end of installation.
When I put him alone worked just fine.
Thanks!!!!

Btw, your dotfiles is awesome. :)

**Reply by Matt-FTW** [reply]
>  For some reason the "ttf-ms-win11-auto" package gets an error in the end of installation. When I put him alone worked just fine.

Umm thats weird, dont know why that happens honestly xd

> Btw, your dotfiles is awesome. :)

Thanks buddy!, im glad you like them :)
